uint8 count
float32[] lineEqParam
float32[] centerGravity